src/app/pages/office/office.reducer.ts

```diff
@@ -0,0 +1,16 @@
+import * as officeActions from './office.actions';
+import { OfficeModel } from './office.model';
+
+export const initialState: OfficeModel = {
+  dangerousFaultComments: [],
+  drivingFaultComments: [],
+  seriousFaultComments: [],
+};
+
+export function officeReducer(state = initialState, action: officeActions.OfficeActionTypes): OfficeModel {
```

Comment:
```officeReducer``` is unused in DES3, think we should delete this file?

Reply:
agreed. removed